.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 python:
 - '2.7'
-env:
-- TOXENV=docs
-- TOXENV=py27
+- '3.5'
```

Comment:
**TODO**: make sure we can build the docs again.

Reply:
:+1: